src/config/delegates.json

```diff
@@ -180,22 +180,23 @@
     "description": "Hey, I am a member of the Celo Guardians (formerly known as 'CGP Editors'). I help maintain the history of governance proposals and am an active contributor to the Celo community."
   },
   "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9": {
-    "name": "ReFiCol4🌎",
+    "name": "Celo Colombia Validator🇨🇴",
     "address": "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9",
-    "logoUri": "/logos/validators/refi-col.jpg",
+    "logoUri": "/logos/validators/celo-colombia.png",
     "date": "2024-05-01",
     "links": {
-      "website": "https://linktr.ee/reficolombia",
-      "twitter": "https://twitter.com/reficolombia"
+      "website": "https://linktr.ee/celocolombia",
+      "twitter": "https://twitter.com/celo_col"
     },
     "interests": [
-      "ReFi",
+      "Regional HUBs",
       "Community",
       "Transparency",
       "Accountability",
-      "Good Vibes"
+      "Good Vibes",
+      "ReFi"
```

Comment:
```
{
    "code": "too_big",
    "maximum": 5,
    "type": "array",
    "inclusive": true,
    "exact": false,
    "message": "Array must contain at most 5 element(s)",
    "path": [
      "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9",
      "interests"
    ]
  }
```

That's the error on preview

we need to either relax the zod rules or remore an items